lint.js

```diff
@@ -64,7 +65,13 @@ More information: https://speccy.io/rules/1-rulesets#${rule.name}
     return output;
 }
 
-const command = async (specFile, cmd) => {
+const validateAsync = (spec, options) => {
```

Comment:
The `validator.validate` method was a little bit tricky to use when validating multiple files. To avoid even more callbacks I wrapped it in a Promise and piped the result to some `map`s and a `reduce`